depthai_bridge/src/DisparityConverter.cpp

```diff
@@ -42,7 +42,7 @@ void DisparityConverter::toRosMsg(std::shared_ptr<dai::ImgFrame> inData, Dispari
     outDispImageMsg.t = _baseline / 100;  // converting cm to meters
     sensor_msgs::msg::Image& outImageMsg = outDispImageMsg.image;
 #else
-    auto rosNow = ros::Time::now();
+    auto rosNow = ::ros::Time::now();
```

Comment:
Use `rosOrigin::Time::now()`?

Reply:
rosOrigin is defined only in `BridgePublisher` because thats where it was most needed.

Reply:
So disparity converter is ros2 only?

Reply:
No. the common function under ros and rclcpp was `ok()` which is called in while function in Bridge Publisher thats the only point where I needed to have a way to address it. 
In rest of the classes I only need Time which is `ros::Time` or `rclcpp.Clock` so there was no need to define `rosOrigin`

Reply:
Sounds good